bokeh/models/glyphs.py

```diff
@@ -893,6 +926,39 @@ class Text(Glyph):
     The %s values for the text.
     """)
 
+class VBar(Glyph):
+    """ Render vertical bars, given a center coordinate, width and (top, bottom) coordinates. """
+
+    __example__ = "tests/glyphs/VBar.py"
+
+    # a canonical order for positional args that can be used for any
+    # functions derived from this class
+    _args = ('x', 'width', 'top', 'bottom')
+
+    x = NumberSpec(help="""
+    The x-coordinates of the centers of the bars.
+    """)
+
+    width = NumberSpec(help="""
+    The widths of the vertical bars.
+    """)
+
+    bottom = NumberSpec(help="""
+    The y-coordinates of the bottom edges.
+    """)
```

Comment:
I'm not quite sure about this. What's the intended use case for VBar and HBar? Is likely isn't a user passing in data, in which case I think it makes more sense for the user to be explicit about what the top and bottom values should be, would that be right?